### DISCUSSION TITLE
Dependency Dashboard sorts alphabetically instead of numerically

### DISCUSSION THREAD

**Discussion by mannycarrera4** [renovatebot/renovate#10292] [Request Help]
Is it possible for renovate to list the dependencies numerically instead of alphabetically?

```chore(deps): update dependency ts-loader to v7
chore(deps): update dependency ts-loader to v8
chore(deps): update dependency ts-loader to v9 <--- in increasing order

chore(deps): update dependency dotenv to v10
chore(deps): update dependency dotenv to v9   <--- in decreasing order
...
chore(deps): update dependency ts-node to v10
chore(deps): update dependency ts-node to v9  <--- in decreasing order```

**Comment by rarkins** [comment]
Can you create a reproduction repo to demonstrate?

**Reply by mannycarrera4** [reply]
Will try!

**Reply by mannycarrera4** [reply]
@rarkins  I was able to repo here: https://github.com/mannycarrera4/manny-sketches-site/issues/12

This happens when things jump a major version from single to double digit example emotion core. It went from v9 -> v10 and v10 is shown first

**Reply by rarkins** [reply]
Where exactly should I be looking?

**Reply by mannycarrera4** [reply]
Hey @rarkins  my apologies, here's the updated dep dashboard, notice how dotenv v10 comes before v9: https://github.com/mannycarrera4/manny-sketches-site/issues/12

**Reply by rarkins** [reply]
Is this a problem except for the rare case where both:
- separateMultipleMajor is enabled, and
- there is both a v9 and v10 upgrade available?

**Reply by mannycarrera4** [reply]
correct, and it's not an a bug, but figured it would be a nice to have, thoughts?
